app/src/tests/integration/IntegrationTest.ts

```diff
@@ -1,21 +1,5 @@
-import {createAsyncThunk} from "@reduxjs/toolkit";
-import {CurrentUser, CustomAsyncThunkOptions, UserLoginDetailProps} from "../../types";
-import {LoginMutationType} from "../../app/store/login/loginThunks";
-import {api} from "../../apiService";
-import store from "../../app/store/store";
-
-export const login = async (): Promise<CurrentUser> => {
-
-    const testUser = {email: "test.user@example.com", password: "password"};
-    const dispatch = jest.fn();
-    const fetchTokenThunk = createAsyncThunk<Record<string, any>, UserLoginDetailProps, CustomAsyncThunkOptions>(
-        LoginMutationType.GetToken,
-        async (prop, thunkAPI) => api(store)
-            .postAndReturn("/auth/login", prop, thunkAPI));
-    const fetchTokenAction = fetchTokenThunk(testUser);
-    await fetchTokenAction(dispatch, jest.fn(), jest.fn());
-    const result = dispatch.mock.calls[1][0];
-    expect(result["type"]).toBe("GetToken/fulfilled");
-
-    return result["payload"];
+// Long-lived JWT token for test backend configuration which we can use to authenticate in integration tests
+export const testUser = {
```

Comment:
The integration tests previously used the now-retired basic auth login to access protected endpoints - we don't yet have a full integration test solution for the Github login (and doing a full login might not be appropriate for testing non-auth functionality), so here I've just replaced that with a long-lived (20 year) JWT token generated from the token issuer code.